src/types/checkbox.js

```diff
@@ -20,6 +21,8 @@ export default multiple({
                     value   : opt.value,
                     checked : opt.selected,
 
+                    required : req,
```

Comment:
`required : req` vs `required: options.req` doesn't seem consistent amongst types currently.

Let's make them all `required: options.req` since nothing seems to ever use that `req` for anything else.